x/staging/subspaces/keeper/keeper.go

```diff
@@ -84,7 +84,7 @@ func (k Keeper) AddAdminToSubspace(ctx sdk.Context, subspaceID, user, owner stri
 
 	// Check if the user we want to set as admin is already an admin
 	if k.IsAdmin(ctx, subspaceID, user) {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "the user with address %s is already an admin", user)
+		return sdkerrors.Wrapf(types.ErrAlreadySubspaceAdmin, user)
```

Comment:
Can we rename this to be `ErrDuplicatedAdmin` to make it similar to other errors?